submissions/cynthiaNwakaeme/CSS/style.css

```diff
@@ -0,0 +1,286 @@
+*,body{
+    font: 15px/1.5, Arial, Helvetica, sans-serif;
+    padding: 0;
+    margin: 0;
+  
+}
+
+.nav-container{
+    width: 95%;
+    margin: auto;
+    overflow: hidden;
+}
+ul{ 
+    padding: 0;
+    margin: 0;
+}
+/* THE HEADER */
+header{
+    background:#9d426b; 
+    color: white;
+    padding-top: 10px;
+    min-height: 50px;
+    border-bottom: goldenrod 3px solid;
+    
+}
+header a{
+    color: white;
+    text-decoration: none;
+    font-size: 16px;
+}
+
+header li{
+    display: inline;
+    padding:0 30px;
+}
+header #nav-branding{
+    float: left;
+    
+}
+header #nav-branding h2,header #nav-branding p{
+    margin:0;
+    padding:0;
+    color:white;
+}
+header nav{
+    float: right;
+    margin-top: 15px;
+}
+header a:hover{
+    text-transform: uppercase;
+    color:white;
+    font-weight: bold;
+
+}
+  
+/* THE SIGN_UP FORM INFORMATION */
+.container{
+    overflow: hidden;
+    padding: 30px 0;
+    margin: auto;
+}
+
+        /* MAIN FRAME */
+#main{
+    float:left;
+    text-align: left;
+    width: 40%;
+    margin: 50px 80px 0 80px;
+}
+#main h1{
+    color: #9d426b;
+    font-size: 35px;
+}
+#main p{
+    color: rgb(146, 144, 144);
+    font-size: 25px;
+    padding: inherit;
+}
+#main p a{
+    color: #9d426b;
+    font-size: 22px;
+}
+        /* ASIDE FRSME */
+#aside{
+    float: left;
+    width: 40%;   
+}
+#aside fieldset{
+    border:1px solid rgb(129, 129, 129);
+    border-radius: 3%;
+    margin-bottom: 10px;
+    width: 100%;
+}
+#aside .form-details{
+    padding: 20px;
+    margin-left: 20px;
+}
+.form-details label{
+    color: rgb(146, 144, 144);
+    font-size: 13px;
+    margin-bottom: 20px;     
+}
+.form-details input[type="text"],
+.form-details input[type="tel"],
+.form-details input[type="email"],
+.form-details input[type="password"]
```

Comment:
I think it's better to have `input` field targeted instead of this grouping. I noticed your button is a input stage, you can modify it to be `button` instead.

Reply:
@ToluAkin .
Thank you for your prompt response.
i used this method to target the changes on the `input` because the terms and condition has an `input[type=checkbox]` to agree to TandC.
Please suggestions on how i can make this better is welcomed.

Thank you ma.

-

Reply:
Oh! It is fine then.

Reply:
Hello @ToluAkin I have worked on all stated issues. Kindly approve